CHANGELOG.md

```diff
@@ -2,6 +2,62 @@
 
 ## Unreleased
 
+### API
+
+* Fix ConfigUtil#getString ConcurrentModificationException
+  ([#6841](https://github.com/open-telemetry/opentelemetry-java/pull/6841))
+
+### SDK
+
+#### Traces
+
+* Stabilize ExceptionEventData
+  ([#6795](https://github.com/open-telemetry/opentelemetry-java/pull/6795))
+
+#### Metrics
+
+* Stabilize metric cardinality limits
+  ([#6794](https://github.com/open-telemetry/opentelemetry-java/pull/6794))
+* Enable reusuable_data memory mode by default
+  ([#6799](https://github.com/open-telemetry/opentelemetry-java/pull/6799))
```

Comment:
maybe move to Exporters section and list which specific exporters (OTLP, Prometheus?) it affects?